models/ticket_history/zendesk_field_history_pivot.sql

```diff
@@ -44,7 +44,7 @@ with field_history as (
 
     select 
         ticket_id,
-        cast({{ dbt_utils.dateadd('day', 0, 'valid_starting_at') }} as date) as date_day
+        cast({{ dbt_utils.dateadd('day', 0, 'valid_starting_at') }} as date) as valid_from
 
         {% for col in results_list if col in var('ticket_field_history_columns') %}
         {% set col_xf = col|lower %}
```

Comment:
everything looks good! but was wondering about line 51 -- why a `min` instead of `max`? @fivetran-kristin

Reply:
Reviewed in person. @fivetran-jamie can you approve so that I can merge please?